packages/react-vapor/tsconfig.json

```diff
@@ -11,5 +11,5 @@
         "noUnusedLocals": true,
         "lib": ["es2015", "dom"]
     },
-    "exclude": ["node_modules", "dist", "coverage", "*.js"]
+    "exclude": ["**/node_modules", "**/dist", "**/coverage", "*.js", "**/target"]

```

Comment:
is that in the PR scope ?
just wondering

Reply:
Probably not, if we were being 100% strict on the scope of PRs.